src/components/BorrowerProfile/CommentsAndWhySpecial.vue

```diff
@@ -98,9 +98,14 @@
 								<h3>
 									{{ comment.authorName }}
 								</h3>
-								<!-- TODO add team name <h4>
-									Team Name
-								</h4> -->
+								<h4 v-if="comment.lenderTeam">
+									<router-link
+										:to="`/team/${comment.lenderTeamPublicId}`"
```

Comment:
Since `lenderTeamPublicId` can be `null`, should the `v-if` also check that prop?